web-bluetooth/write-descriptor-async-await.html

```diff
@@ -0,0 +1,50 @@
+---
+feature_name: Web Bluetooth / Characteristic User Description (Async Await)
+chrome_version: 58
+check_min_version: true
+feature_id: 5264933985976320
+icon_url: icon.png
+index: index.html
+---
+
+{% include_relative _includes/intro.html %}
+
+<p>This sample illustrates the use of the Web Bluetooth API to change
+Characteristic User Description characteristic's descriptor of a service from a
```

Comment:
... to write to the descriptor 'Characteristic User Description' on a nearby Bluetooth ...